### PR TITLE
webpack-cli: new init documentation

### PR DESCRIPTION
**Pull request by rishabh3112**
_describe your changes..._
PR for #4741

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -44,13 +45,45 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
...
+| `init`       | `init\|c [generation-path] [options]`     | Initialize a new webpack project.                                               |
 | `loader`     | `loader\|l [output-path]`                 | Scaffold a loader.                                                              |
 | `plugin`     | `plugin\|p [output-path]`                 | Scaffold a plugin.                                                              |
 | `serve`      | `serve\|s [options]`                      | Run the `webpack-dev-server`.                                                   |
 | `version`    | `version\|v [commands...]`                | Output the version number of `webpack`, `webpack-cli` and `webpack-dev-server`. |
 | `watch`      | `watch\|w [entries...] [options]`         | Run webpack and watch for files changes.                                        |
 
+### Init
+
+Used to initialize new webpack project
```
Comment (chenxsan) [inline src/content/api/cli.md:57]:
```suggestion
Used to initialize a new webpack project
```

File: src/content/api/cli.md [hunk 2]
```diff
@@ -44,13 +45,45 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
...
+
+**example**
+
+```bash
+npx webpack init ./my-app --force --template=default
+```
+
+#### Generation Path
+
+Location of where to generate configuration. defaults to `process.cwd()`
```
Comment (chenxsan) [inline src/content/api/cli.md:71]:
```suggestion
Location of where to generate the configuration. Defaults to `process.cwd()`.
```

File: src/content/api/cli.md [hunk 3]
```diff
@@ -44,13 +45,45 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
...
+
+`string = default`
+
+Name of template to generate.
+
+**`--force`**
+
+`boolean`
+
+To generate project without questions. If passed, default answer for each question will be used.
```
Comment (chenxsan) [inline src/content/api/cli.md:85]:
```suggestion
To generate a project without questions. When enabled, default answer for each question will be used.
```

File: src/content/contribute/index.md [hunk 4]
```diff
@@ -45,7 +45,6 @@ The remainder of this section of the site is dedicated to developers such as you
 
 - [Writing a Loader](/contribute/writing-a-loader)
 - [Writing a Plugin](/contribute/writing-a-plugin)
-- [Writing a Scaffold](/contribute/writing-a-scaffold)
```
Comment (chenxsan) [inline src/content/contribute/index.md:48]: Let's add a redirection for this in `webpack.ssg.js`, as there might be links somewhere to it, e.g.:

```js
      new RedirectWebpackPlugin({
        redirects: {
         '/contribute/writing-a-scaffold': '/api/cli/#init',
        }
    })
```
Same applied to the removed `src/content/guides/scaffolding.md`.

Comment (rishabh3112) [reply src/content/contribute/index.md:48]: Did the same and fixed other grammatical errors. Thanks.

**Review by chenxsan — APPROVED** [review]

File: src/content/api/cli.md [hunk 5]
```diff
@@ -44,13 +45,45 @@ webpack-cli offers a variety of commands to make working with webpack easy. By d
...
+Used to initialize a new webpack project.
+
+```bash
+npx webpack init [generation-path] [options]
+```
+
+**example**
+
+```bash
+npx webpack init ./my-app --force --template=default
```
Comment (snitin315) [inline src/content/api/cli.md:66]: as `--force` will use the default.

```suggestion
npx webpack init ./my-app --force
```

Comment (rishabh3112) [reply src/content/api/cli.md:66]: I want to explain all possible way to use command in a single example, which includes specifying template.

**Review by snitin315 — APPROVED** [review]

**Pull request merged**